.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should the `.idea` directory and its files be included or git-ignored?

Reply:
I deleted the files related to idea and added them to git-ignored